lib/Doctrine/DBAL/Platforms/PostgreSQL100Platform.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Doctrine\DBAL\Platforms;
+
+use Doctrine\DBAL\Platforms\Keywords\PostgreSQL100Keywords;
+
+/**
+ * Provides the behavior, features and SQL dialect of the PostgreSQL 10.0 database platform.
+ *
+ * @link   www.doctrine-project.org
+ * @since  2.6
+ */
+class PostgreSQL100Platform extends PostgreSQL94Platform
+{
+	/**
+	 * {@inheritdoc}
+	 */
+	protected function getReservedKeywordsClass() : string
+	{
+		return PostgreSQL100Keywords::class;
+	}
+
+	public function getSequenceMetadataSQL(string $rawSequenceName, string $schemaName) : string
```

Comment:
There shouldn't be a semantic difference between this method and its parents from `AbstractPlatform` and `PostgreSqlPlatform`. Otherwise, the caller which depends on the abstract name will not know whether it should pass raw or non-raw sequence name.